run_optimization.py

```diff
@@ -96,19 +96,26 @@ def main(sweep_config_path, num_trials=1):
         Number of trials to run. This will add num_trials to the existing trials in the sweep_results.db
     """
 
+    # Change to the sweep directory
```

Comment:
Porbably want a context manager to handle switching directories for you. [See here for an example](https://stackoverflow.com/questions/431684/equivalent-of-shell-cd-command-to-change-the-working-directory/431747#431747). Or [this answer](https://stackoverflow.com/questions/41742317/how-can-i-change-directory-with-python-pathlib) on how to create your own context manager using a decorator. I'd try the first one first.

Reply:
That's cool, didn't know you could do that. I foresee using this functionality elsewhere so I will put it in its own module (maybe to be expanded in the future).